app/common/services/data-export.service.js

```diff
@@ -26,6 +26,10 @@ function DataExport($rootScope, ConfigEndpoint, ExportJobEndpoint, $q, _, $windo
             loadingStatus(false, err);
         });
     }
+    function cancelExport(jobId) {
+        //TODO: Add cancel-job
```

Comment:
We will need to define this but this is ok for now.

Reply:
Yes, that will happen in the polling-pr...